src/core/frames/frame_controller.ts

```diff
@@ -72,9 +73,15 @@ export class FrameController implements AppearanceObserverDelegate, FetchRequest
     }
   }
 
-  async loadResponse(response: FetchResponse) {
+  async loadResponse(fetchResponse: FetchResponse) {
+    if (fetchResponse.redirected) {
+      this.listenForChangestoSrc = false
```

Comment:
I'm dissatisfied with toggling the `listenForChangesToSrc` private property, but it illustrates the point and helps pass the suite.

When responding to successful form submissions, the `Navigator` proposes a visit.

https://github.com/hotwired/turbo/blob/8bce5f17cd697716600d3b34836365ebcdc04b3f/src/core/drive/navigator.ts#L86-L88

Unfortunately, Frames don't have the same access to a Navigator instance at the moment, so our options are limited.